installing/installing_openstack/preparing-to-install-on-openstack.adoc

```diff
@@ -0,0 +1,30 @@
+[id="preparing-to-install-on-openstack"]
+= Preparing to install on OpenStack
+include::modules/common-attributes.adoc[]
+:context: preparing-to-install-on-openstack
+
+toc::[]
+
+[id="preparing-to-install-on-openstack-prerequisites"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+
+[id="choosing-an-method-to-install-ocp-on-openstack"]
+== Choosing a method to install {product-title} on OpenStack
+
+You can install {product-title} on {rh-openstack-first} by using one of the following deployment methods.
+
+* **xref:../../installing/installing_openstack/installing-openstack-installer-custom.adoc#installing-openstack-installer-custom[Installing a cluster on OpenStack with customizations]**: You can install a customized cluster on {rh-openstack}. The installation program allows for some customization to be applied at the installation stage. Many other customization options are available xref:../../post_installation_configuration/cluster-tasks.adoc#post-install-cluster-tasks[post-installation].
+
+* **xref:../../installing/installing_openstack/installing-openstack-installer-kuryr.adoc#installing-openstack-installer-kuryr[Installing a cluster on OpenStack with Kuryr]**: You can install a customized {product-title} cluster on {rh-openstack} that uses Kuryr SDN. Kuryr and {product-title} integration is primarily designed for {product-title} clusters running on {rh-openstack} VMs. Kuryr improves the network performance by plugging {product-title} pods into {rh-openstack} SDN. In addition, it provides interconnectivity between pods and {rh-openstack} virtual instances.
+
+* **xref:../../installing/installing_openstack/installing-openstack-user.adoc#installing-openstack-user[Installing a cluster on OpenStack on your own infrastructure]**: You can install {product-title} on {rh-openstack} that runs on user-provisioned infrastructure. By using this installation method, you can integrate your cluster with existing infrastructure and modifications. For user-provisioned installations, you must create all {rh-openstack} resources, like Nova servers, Neutron ports, and security groups. You can use the provided Ansible playbooks to assist with the deployment process.
```

Comment:
Nit: "user-provisioned installations", seems like a new term.  Could it be "For installations on user-provisioned infrastructure, you must..."

Or, it really is a term that is new to me.